### PR TITLE
510-minify-option-packages-dump

### PR DESCRIPTION
**Pull request by cfv1000**
add minification option to the satis configuration

### CODE REVIEW

File: tests/Builder/PackagesBuilderDumpTest.php [hunk 1]
```diff
@@ -83,7 +83,8 @@ public function testNominalCase($providers = false)
                 $packageName = key($arrayPackages);
                 $arrayPackages[$packageName][$i . '.0']['uid'] = 1;
                 $hash = current($packagesJson['providers'][$packageName]);
-                $includeJson = str_replace(['%package%', '%hash%'], [$packageName, $hash], $packagesJson['providers-url']);
+                $includeJson = str_replace(['%package%', '%hash%'], [$packageName, $hash],
+                    $packagesJson['providers-url']);
```
Comment (staabm) [inline tests/Builder/PackagesBuilderDumpTest.php]: unrelated change?

Comment (alcohol) [reply tests/Builder/PackagesBuilderDumpTest.php]: Yeah, I think your editor did something quirky here. Can you undo this?

File: res/satis-schema.json [hunk 2]
```diff
@@ -284,6 +284,10 @@
         "_comment": {
             "type": ["array", "string"],
             "description": "A key to store comments in"
+        },
+        "pretty-print": {
+            "type": "boolean",
+            "description": "Defaults to true. If true, the output files will NOT be minified (will look pretty tough)."
```
Comment (alcohol) [inline res/satis-schema.json]: I think this description is a bit misleading. Minification is not really the same as simply having no formatting whitespace.

Comment (cfv1000) [reply res/satis-schema.json]: updated

File: tests/Builder/PackagesBuilderDumpTest.php (hunk 1 above)
Comment (cfv1000) [reply tests/Builder/PackagesBuilderDumpTest.php]: undone

**Comment by alcohol** [comment]
Thanks
